### PR TITLE
[FIX][connector] Migration to be odoo master compliant

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
All unit and automatic tests defined in module are OK after modification.

### CODE REVIEW

**Comment by guewen** [comment]
Thanks for the effort! There is one test that do not pass.

**Comment by bwrsandman** [comment]
Could you update your travis file to match the one in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml

**Comment by rvalyi** [comment]
:+1:  awesome, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,21 @@
...
-# Need flake8 for pep8 testing
-# Manually get PyChart
-# Install tested version of odoo (official or ocb)
-# Get modules from other repos which have dependencies (in this case travel requires modules from lp:partner-contact-management and lp:openerp-hr
-install: 
-    - pip install coveralls flake8
-    - pip install http://download.gna.org/pychart/PyChart-1.39.tar.gz
-    - pip install ${ODOO}
+env:
+ - VERSION="8.0"
```
Comment (bwrsandman) [inline .travis.yml]: OCB and odoo, from sample files in quality-tools

**Comment by bwrsandman** [comment]
:+1:

**Comment by lmignon** [comment]
Hi,

Everything is not yet fixed for V8... Server starts without error, Travis is green but at runtime, call to 'delay' fails. I'll fixes remaining problems ASAP.

lmi

**Comment by bwrsandman** [comment]
:+1:

**Comment by lmignon** [comment]
Hi, 

At this stage, delayed jobs works on V8 in both modes (simple vs multi workers)

Regards,

lmi

File: connector/unit/mapper.py [hunk 2]
```diff
@@ -660,9 +660,14 @@ def _apply_with_options(self, map_record):
         result = {}
         for from_attr, to_attr in self.direct:
             if callable(from_attr):  # in a modifier
-                # the name of the attribute is the first arg of the
-                # closure
-                attr_name = from_attr.__closure__[0].cell_contents
+                # the name of the attribute is the first arg the original
+                # function. BUT the argument order seems to be not enforced
+                # by python in the closure so we use the first non callable
+                # cell_contents in the closure as attr_name
```
Comment (guewen) [inline connector/unit/mapper.py:666]: You are right! Interesting post about that: http://stackoverflow.com/a/17145475
It deserves to be backported.

**Comment by guewen** [comment]
Excellent work!

Thanks :heart: 

:+1:

**Pull request merged**